5-analysis/analysis-powder-diffraction.ipynb

```diff
@@ -0,0 +1,1665 @@
+    "\n",
+    "#### Exercise 5.1: Set Fit Parameters\n",
+    "\n",
+    "Select the parameters to be refined during the fitting process.\n",
+    "\n",
+    "**Hint:** You can start with the same parameters as in the Si fit, but\n",
+    "this time you will refine the scale factor of the LBCO phase and the\n",
+    "background points for the second simulation.\n",
+    "\n",
+    "**Solution:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "72ba7c10",
+   "metadata": {},
+   "source": [
+    "project_2.experiments[\"sim_lbco\"].linked_phases[\"lbco\"].scale.free = True\n",
+    "for line_segment in project_2.experiments[\"sim_lbco\"].background:\n",
+    "    line_segment.y.free = True"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76151a8e",
+   "metadata": {},
+   "source": [
+    "#### Exercise 5.2: Run Fitting\n",
+    "\n",
+    "Visualize the measured and calculated diffraction patterns before fitting and\n",
+    "then run the fitting process.\n",
+    "\n",
+    "**Hint:** Use the `plot_meas_vs_calc` method of the project to visualize the\n",
+    "measured and calculated diffraction patterns before fitting. Then, use the\n",
+    "`fit` method of the `analysis` object of the project to perform the fitting\n",
+    "process.\n",
+    "\n",
+    "**Solution:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "17d6fab3",
+   "metadata": {},
+   "source": [
+    "project_2.plot_meas_vs_calc(expt_name=\"sim_lbco\")"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "code",
+   "id": "cab87e11",
+   "metadata": {},
+   "source": [
+    "project_2.analysis.fit()"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e8311ca",
+   "metadata": {},
+   "source": [
+    "#### Exercise 5.3: Find the Misfit in the Fit\n",
+    "\n",
+    "Visualize the measured and calculated diffraction patterns after the fit. As\n",
+    "you can see, the fit shows noticeable discrepancies. If you zoom in on different regions of the pattern,\n",
+    "you will observe that all the calculated peaks are shifted to the left.\n",
+    "\n",
+    "What could be the reason for the misfit?\n",
+    "\n",
+    "**Hint**: Consider the following options:\n",
+    "1. The conversion parameters from TOF to d-spacing are not correct.\n",
+    "2. The lattice parameters of the LBCO phase are not correct.\n",
+    "3. The peak profile parameters are not correct.\n",
+    "4. The background points are not correct.\n",
+    "\n",
+    "**Solution**:\n",
+    "1. ❌ The conversion parameters from TOF to d-spacing were set based on the data reduction process and this was verified in the Si fit.\n",
+    "2. ✅ The lattice parameters of the LBCO phase were set based on the CIF data, which is a good starting point, but they are not necessarily as accurate as needed for the fit. The lattice parameters may need to be refined.\n",
```

Comment:
I'm not sure if students will get this right. We just give them the parameters without any further explanation or qualification. They have no reason to think that the parameters are inaccurate. To fix this, it may be enough to explain that the parameters are a first estimate for the LBCO crystal structure.

Reply:
Thanks for the suggestion. I’ve added a note where the LBCO structure is introduced, clarifying that the structural parameters provided are not necessarily the most accurate, but serve as a reasonable starting point for the fit.